packages/astro/test/fixtures/third-party-astro/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "dependencies": {
-    "astro": "^1.0.0",
+    "astro": "workspace:*",
```

Comment:
The 1.0.0 was used to bypass the peer dep warning, but I found this test to use the old 1.0 code instead, breaking the compiler upgrade as both 1.0 and 2.0 code exist at the same time.